fml/BUILD.gn

```diff
@@ -253,6 +253,10 @@ executable("fml_unittests") {
     "$flutter_root/runtime:libdart",
     "$flutter_root/testing",
   ]
+
```

Comment:
The other tests and binaries pull in the dependency via skia or dart.  Since chinmay severed the dependency on both, this is explicitly required now